Governance/tech/drs.md

```diff
@@ -1,6 +1,7 @@
 # Choosing a directory structure (DRS) and filenames
 
-The names you choose for files and directories and generally the way you organise your data, i.e. your directory structure (DRS) can help navigating the data and provide extra information, avoid confusion and the user ending up accessing the wrong data. As for many other cases the best file organisation will depend on the specific research project and the actual server where the data is stored. Here we are just listing a few guidelines and tips to help you decide.  
+The names you choose for files and directories and generally the way you organise your data, i.e. your directory structure can help navigating the data and provide extra information, avoid confusion and the user ending up accessing the wrong data. As for many other cases the best file organisation will depend on the specific research project and the actual server where the data is stored. The global climate modelling intercomparison project has adopted a Data Reference Syntax (DRS), based on the controlled vocabularies (CVs) used in model metadata, to define their file names and directory structures. 
```

Comment:
Perhaps say CMIP too (since most people know it)

Reply:
I actually thought not to so that the (ACRONYMS) were for things I want to draw attention to (DRS, CV), but I could (should?) embolden those words instead/as well.